### ISSUE TITLE
hltv.org

### ISSUE THREAD

**Issue by kasper93** [uBlockOrigin/uAssets#500]
### URL(s) where the issue occurs

`hltv.org`

### Describe the issue

Ads everywhere.  Background images, multiple banners on main page. Also you can open currently live match which is also full of ads.

#### Before
![screen shot 2017-07-07 at 22 03 55](https://user-images.githubusercontent.com/1126053/27974892-3f95b2d0-6360-11e7-92c8-eae3dd9e1485.png)
#### Expected (After)
![screen shot 2017-07-07 at 22 18 17](https://user-images.githubusercontent.com/1126053/27975385-3573837a-6362-11e7-86c4-23c5706324e7.png)

### Filters needed
```
@@||static.hltv.org/images/*/AD.gif$image
||gainskins.com$subdocument,domain=www.hltv.org
||hltv.org/retina*.php^$xmlhttprequest
hltv.org##.rek
hltv.org##.col-rek
hltv.org##.betting
hltv.org##.csgofastbetting
hltv.org##.three-quarter-width:has(> .betting)
hltv.org##[hidden] + .section-spacer
hltv.org##a[href^="http://egbaffiliates.com/track"]
hltv.org##.newstext-con > p:last-child > span:only-child > br:only-child
hltv.org##body:style(background: #e5e7ea !important;)
```

### Settings

- Easylist + Easyprivacy + POL + Fanboy’s Annoyance

**Comment by kasper93** [comment]
@okiehsch: Thanks for merge. There is still banner on top of the ranking page. `https://www.hltv.org/ranking/teams/2017/july/3`

**Comment by okiehsch** [comment]
Thanks, I looked for a reason to add your suggested filter
`hltv.org##a[href^="http://egbaffiliates.com/track"]`
but somehow missed this pretty obvious one.

**Comment by okiehsch** [comment]
I already got some feedback that this filter
`hltv.org##.three-quarter-width:has(> .betting)`
which I thought was essentially an ad, prevents the betting on game pages and this is apparently 
a core functionality for some, so I will remove it.

**Comment by kasper93** [comment]
Sure, I would say this is 90% an ad and 10% functionality but if ppl wants this, ok.

You need to remove 
```
hltv.org##a[href^="http://egbaffiliates.com/track"]
```
because it will block egb.com ~~advert~~ functionality in betting box. And to not break ranking page again probably hiding image banner will be fine.
```
||hltv.org/img/static/egbranking.png
```

**Comment by kasper93** [comment]
@okiehsch: They've updated the page. 

```
||geo3.hltv.org/*.php$xmlhttprequest
hltv.org##.col-rek2
```

**Comment by ptkovsky** [comment]
Hello. There is a continuing weird problem with this site's ads not being blocked. Not really sure if need to make a new issue or this is fine too, but can you take a look at this? https://github.com/el1t/uBlock-Safari/issues/65 Maybe something can be done for this site just playing with filter syntax?

**Comment by kasper93** [comment]
@ptkovsky: I suggest you enabling `Adguard Base Filters`. It contains latest filters for this site. 

Filters could be backported to uBO list if maintainers decide to do so.

**Comment by okiehsch** [comment]
If somebody comes up with a working filter that fixes https://github.com/el1t/uBlock-Safari/issues/65 for Safari users I will consider adding it to uBO-filters, but I do not have any macOS device , so I can't help.

**Comment by ptkovsky** [comment]
Thanks for your interest in the problem. Yeah, enabling Adguard filters definitely helps as it cuts most of the ads on the main page except the middle banner at the top.


We'll see how it goes, maybe @el1t will figure something out on his side of the things. Personally I would like to opt for a more elegant solution rather than adding another 10k filter list just for hiding a few gifs on one page considering that other browsers doesn't have this issue.

**Comment by gorhill** [comment]
Reportedly the set background color is interfering with dark mode: <https://www.reddit.com/r/firefox/comments/b4oq80/firefox_forces_the_body_to_be_white/>.

**Comment by Hrxn** [comment]
rel.
#3025 

This is not working anymore, at least for me..

Background is all black. Suggestion in https://github.com/uBlockOrigin/uAssets/issues/3025#issuecomment-407837216 does not work for me either, I guess site has changed again?

This filter works for me:
`hltv.org##body:style(background-color: #e5e7ea !important;)`

Not sure about that dark mode issue then, though..

**Comment by P0Rtvine** [comment]
`https://www.hltv.org/`

### Screenshot(s)
<details>
<summary>Screen</summary>

![2021-03-30_185726](https://user-images.githubusercontent.com/57287308/113019284-180dbd00-918a-11eb-917d-ea1029e6e92e.png)
![2021-03-30_185800](https://user-images.githubusercontent.com/57287308/113019286-193eea00-918a-11eb-819b-811c0c46ba71.png)
![2021-03-30_185644](https://user-images.githubusercontent.com/57287308/113019287-193eea00-918a-11eb-8605-d9d04903186e.png)
</details>

**Comment by okiehsch** [comment]
@P0Rtvine I can't reproduce.
![image](https://user-images.githubusercontent.com/16838044/113094741-89815600-91f2-11eb-9555-f00010fa75de.png)

**Comment by P0Rtvine** [comment]
@okiehsch
And what to do?

**Comment by mapx-** [comment]
test

```
hltv.org##[href*="utm_source"]
hltv.org##[href*="&affid"]
```

**Comment by P0Rtvine** [comment]
leftover
### Screenshot(s)
<details>
<summary>Screen</summary>

![2021-03-31_133814](https://user-images.githubusercontent.com/57287308/113131873-6a9cb700-9226-11eb-9908-084bfd66193d.png)
</details>

**Comment by mapx-** [comment]
updated above

**Comment by P0Rtvine** [comment]
too much
![2021-03-31_135741](https://user-images.githubusercontent.com/57287308/113134175-29f26d00-9229-11eb-9d90-f2710d8baa32.png)

**Comment by mapx-** [comment]
well, above like before, if I cannot reproduce nothing to do further

**Comment by P0Rtvine** [comment]
ok, you add this?

> hltv.org##[href*="utm_source"]
> hltv.org##[href*="&affid"]

**Comment by P0Rtvine** [comment]
@mapx- 
When accessing from Russia
### Screenshot(s)
<details>
<summary>Screen</summary>

![2021-03-31_212500](https://user-images.githubusercontent.com/57287308/113192746-d3585380-9267-11eb-8bfb-e91dd9ebb387.png)
![2021-03-31_212524](https://user-images.githubusercontent.com/57287308/113192752-d4898080-9267-11eb-8477-f7803c064e83.png)
![2021-03-31_212541](https://user-images.githubusercontent.com/57287308/113192753-d5221700-9267-11eb-81ec-0be1a0d95f5e.png)
</details>

Proposed cosmetic rules:  "leftover are also hidden"

> hltv.org##[class*="ASdvuasdr"]
> hltv.org##a[href*=".ashx"]
> hltv.org##.logoCon > div

**Comment by agam3r** [comment]
ads

**Comment by agam3r** [comment]
I encountered ads on `https://www.hltv.org/`.

**Comment by mapx-** [comment]
provide specific page + screenshot

**Comment by Henri-99** [comment]
There are still a number of ads not being blocked on HLTV.

![image](https://user-images.githubusercontent.com/69672231/229106905-af031f77-477d-40d6-9375-827033e25f09.png)

**Comment by mapx-** [comment]
test

```
||hltv.org/imgstatic/*.gif$image
hltv.org##.presented-by-row
hltv.org##.thunderpick-firstcol-box
```
